### PR TITLE
Feature/18916 ess create fss mock api endpoints for cleanup batch folder and respective files

### PR DESCRIPTION
**Pull request by ganpat-tech**
PR contains create Fss mock api endpoints for cleanup batch folder and respective files

### CODE REVIEW

File: UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs [hunk 1]
```diff
@@ -186,5 +186,20 @@ public IActionResult GetBatchStatus([FromRoute, Required] string batchId)
             }
             return Unauthorized();
         }
+
+        [HttpPost]
+        [Route("/cleanUp")]
+        public IActionResult CleanUp([FromBody] List<string> batchId)
+        {
+            if (batchId != null)
```
Comment (janvi13603) [inline UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs:194]: Add count check to if condition
if (batchId != null && batchId.Count > 0)

File: UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs [hunk 2]
```diff
@@ -107,5 +108,21 @@ public BatchStatusResponse GetBatchStatus(string batchId, string homeDirectoryPa
...
             return batchStatusResponse;
         }
+
+        public bool CleanUp(List<string> batchId,  string homeDirectoryPath)
+        {
+            string folderName = fileShareServiceConfiguration.Value.FolderDirectoryName;
+            bool deleteFlag = false;
+            foreach (var item in batchId)
+            {
+                string uploadBlockFolderPath = Path.Combine(homeDirectoryPath, folderName, item);
```
Comment (janvi13603) [inline UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs:118]: Kindly change variable name to exchangeSetZipFolderPath

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/AzureADB2CAuthenticationTests.cs [hunk 3]
```diff
@@ -11,20 +11,27 @@ namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
...
         private ExchangeSetApiClient ExchangeSetApiClient { get; set; }
+        private FssApiClient FssApiClient { get; set; }
         private TestConfiguration Config { get; set; }
-        private string EssB2CToken { get; set; }        
+        private string EssB2CToken { get; set; }
         private string EssB2CCustomizedToken { get; set; }
         public DataHelper DataHelper { get; set; }
-
+        private string FssJwtToken { get; set; }
+        private readonly List<string> cleanUpBatchIdList = new List<string>();
```
Comment (janvi13603) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/AzureADB2CAuthenticationTests.cs:20]: Can casing of "cleanUpBatchIdList" changed to pascal case "CleanUpBatchIdList" in all test case files to maintain consistency.

Also change casing of sinceDateTime to SinceDateTime in current file.

Remove line breaks at 23/24

Comment (ganpat-tech) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/AzureADB2CAuthenticationTests.cs]: Did changes accordingly. Thanks!

File: UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs (hunk 2 above)
Comment (ganpat-tech) [reply UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs]: Did changes accordingly. Thanks!

File: UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs (hunk 1 above)
Comment (ganpat-tech) [reply UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs]: Thanks for highlighting it, making the changes accordingly.

**Review by janvi13603 — APPROVED** [review]
Thanks for fixing review comments.

**Review by barrie-cooper — APPROVED** [review]
LGTM 😃

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**